lib/cbus_elixir/app/meetings.ex

```diff
@@ -5,12 +5,51 @@ defmodule CbusElixir.App.Meetings do
 
   import Ecto.Query
 
+  alias Ecto
   alias CbusElixir.Repo
   alias CbusElixir.App.Meeting
+  alias CbusElixir.Pagination
+  alias CbusElixir.App.Speaker
+  alias CbusElixir.App.Attendee
+
+  @doc """
+  Fetch a paged result for meetings
+  """
+  
+  @spec meetings_for_page_query() :: Ecto.Query
+  def meetings_for_page_query() do
+    
+      from(m in Meeting,
+        join: s in assoc(m, :speakers),
+        where: m.date < ^DateTime.utc_now(),
+        order_by: [desc: m.date],
+        preload: [speakers: s]
+      )
+
+  end
+
+  @doc """
+  Returns the attendees for the given meeting
+  """
+
+  @spec attendees_for_meeting(integer()) :: map()
```

Comment:
This should actually be returning `list(map())` or more specifically `list(Meeting.t())`